easybuild/easyconfigs/b/BemTool/BemTool-20180912.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'Tarball'
+
+name = 'BemTool'
+version = '20180912'
+local_commit = '900479a'
+
+homepage = 'https://github.com/xclaeys/BemTool'
+description = "BemTool is a C++ boundary element library."
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/xclaeys/BemTool/archive/']
```

Comment:
Careful, this is not the BemTool version required by FreeFEM. It needs a special fork, cf. [3rdparty/getall](https://github.com/FreeFem/FreeFem-sources/blob/develop/3rdparty/getall#L201).